<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>606</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="131">
        <red>118</red>
        <green>227</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="131">
        <red>118</red>
        <green>227</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="131">
        <red>118</red>
        <green>227</green>
        <blue>131</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="full" stretch="4,8,0">
     <item>
      <layout class="QHBoxLayout" name="title" stretch="8,2">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Горнолыжный курорт &amp;quot;Игора&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../img/logo_mini.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="line" stretch="0,0,0,0,0,0">
       <item>
        <widget class="QLabel" name="addC">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Добавить клиента&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="name" stretch="1,4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ФИО</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="data" stretch="1,4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Паспортные данные</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passport_line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="birth" stretch="1,4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Дата рождения</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="birth_line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="email" stretch="1,4">
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Адрес</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addres_line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="addres" stretch="1,4">
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>e-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="email_line"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="btn" stretch="6">
       <item>
        <widget class="QPushButton" name="add_btn_2">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>118</red>
               <green>227</green>
               <blue>131</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>118</red>
               <green>227</green>
               <blue>131</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>118</red>
               <green>227</green>
               <blue>131</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
